src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import { SortType } from '../../types/SortType';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  sortField: SortType;
+  setSortField: (field: SortType) => void;
+  todos: Todo[];
+  clearCompleted: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  sortField,
+  setSortField,
+  todos,
+  clearCompleted,
+}) => {
+  const oneTodoCompleted = todos.some(todo => todo.completed);
+
+  const activeTodos = todos.reduce(
+    (acc, todo) => (todo.completed ? acc : acc + 1),
+    0,
+  );
```

Comment:
You can use one reduce method that returns an object with completed and uncompleted.